### PR TITLE
[stable/odoo] Allow to pass arbitrary envs to Odoo

### PR DESCRIPTION
**Pull request by mcanevet**
#### What this PR does / why we need it:
This PR allows to pass arbitrary env to Odoo.

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [X] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [X] Chart Version bumped
- [X] Variables are documented in the README.md
- [X] Title of the PR starts with chart name (e.g. `[stable/mychartname]`)

### CODE REVIEW

File: stable/odoo/values.yaml [hunk 1]
```diff
@@ -59,6 +59,8 @@ odooEmail: user@example.com
 # smtpPassword:
 # smtpProtocol:
 
+env: {}
```
Comment (alemorcuq) [inline stable/odoo/values.yaml]: Can you also add some comments and examples there? For example:

```yaml
## Additional environment variables for Odoo
## env:
##  - name: MY_VAR
##    value: my-value
##
env: []
```

Also note that on the README you said the default value was `[]` and here you put `{}`.

**Review by alemorcuq — CHANGES_REQUESTED** [review]

**Comment by alemorcuq** [comment]
Thank you very much for your PR, @mcanevet. I left a small comment there, otherwise it looks good to me.

File: stable/odoo/values.yaml (hunk 1 above)
Comment (mcanevet) [reply stable/odoo/values.yaml]: Fixed

**Comment by alemorcuq** [comment]
Sorry for the late response, @mcanevet. Thank you for taking your time doing this.

Please, add a comment to the `values.yaml` as I requested on the previous response. And don't forget to sign the DCO too.

Thanks again!

**Comment by alemorcuq** [comment]
Hi, @mcanevet, happy new year. I'm pinging you just in case you want to continue with this PR.

**Comment by alemorcuq** [comment]
Thanks for the update, @mcanevet. The CI is failing some checks:

The Chart version needs a bump to `12.1.0`
```
Old chart version: 12.0.5
New chart version: 11.2.0
Error linting charts: Error processing charts
------------------------------------------------------------------------------------------------------------------------
 ✖︎ odoo => (version: "11.2.0", path: "stable/odoo") > Chart version not ok. Needs a version bump!
```
Link: https://circleci.com/gh/helm/charts/117867?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link

And you also have two commits incorrectly signed off, here's how you can fix it: https://github.com/helm/charts/pull/18581/checks?check_run_id=386602351

**Comment by alemorcuq** [comment]
DCO is still missing, please follow the following steps to fix it: https://github.com/helm/charts/pull/18581/checks?check_run_id=386695726

**Comment by miguelaeh** [comment]
Hi @mcanevet ,
You still need to sign you commit for the PR to pass the tests.

**Pull request closed without merging**